hashintel/hash
# Add page emoji support

## 🌟 What is the purpose of this PR?

Implement `icon` field on `page` entity. So it could be used to identify a page at first glance. 
It can be changed from sidebar, or page title.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1200211978612931/1202700524579012/f) _(internal)_
- [Design](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-App?node-id=1564%3A102859) _(internal)_

## 🐾 Next steps

- Support custom icons as page icons


## ❓ How to test this?

1. Checkout the branch / view the deployment
2. Open any page
3. Try using page icon picker 
4. Page icon should be updating on browser tab, page title and left sidebar

## 📹 Demo

https://user-images.githubusercontent.com/39553853/184602990-3e50d6f3-e150-4401-97e5-fdef6c93840e.mov
